templates/portfolio.html

```diff
@@ -0,0 +1,1059 @@
+{% extends "base.html" %} {% block title %}Portfolio{% endblock %} {% block
+subtitle %}Portfolio{% endblock %} {% block enableLogo %}enable-logo{%
+endblock%} {% block head %}
+<link rel="stylesheet" href="/static/core/css/portfolio.css" />
+{% endblock %} {% block article %}
+<div class="container-full portfolio-full">
+  <img
+    class="octopus"
+    src="/static/core/img/img-portfolio-001.png"
+    aria-hidden="true"
```

Comment:
Done for all images except the background cubes/tentacles, since they are strictly decorative and do not add value for someone using a screen reader. I have left them as aria-hidden="true" to hide them from the accessibility tree: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-hidden_attribute